client/src/components/Common/Heading.vue

```diff
@@ -34,8 +47,15 @@ const element = computed(() => {
 
 <template>
     <div v-if="props.separator" class="separator heading">
-        <div class="stripe"></div>
-        <component :is="element" :class="[sizeClass, props.bold ? 'font-weight-bold' : '']">
+        <b-button v-if="collapsible" variant="link" size="sm" @click="$emit('click')">
+            <icon v-if="collapsed" fixed-width icon="angle-double-down" />
+            <icon v-else fixed-width icon="angle-double-up" />
```

Comment:
Very minor maybe it would it be better to use defined icons like we have been in recent refactors:
```suggestion
            <FontAwesomeIcon v-if="collapsed" fixed-width :icon="faAngleDoubleDown" />
            <FontAwesomeIcon v-else fixed-width :icon="faAngleDoubleUp" />
```

Reply:
Pretty surprised I wrote it this way actually, weird -- I argued against the `icon` wrapper in the past for these.  Happy to make the change, thanks!